CHANGELOG.md

```diff
@@ -2,6 +2,108 @@
 
 Nothing Yet!
 
+
+# Version 0.5.0 (2023-11-27)
+
+This release was probably going to be several releases, but everything got finished at the same time, so here's a Mega Release!
+
+The headline features are:
+
+* New Support For Axo Releases, As An Alternative To Github Releases (Launching Soon™)
+* New Support For Generic Build Steps, In Any Language (Experimental)
+* Significantly Improved MSI Installer Support
+
+## Features
+
+### Axo Releases
+
+Axo Releases users can now enable builtin cargo-dist support by setting `hosting = "axodotdev"` in their
+`[workspace.metadata.dist]`.
+
+Axo Releases has a more robust pipelined model for creating and hosting a release, which more
+closely matches the actual design of cargo-dist. But since we'd only ever supported Github Releases,
+some significant internal reckoning was required.
+
+This reckoning primarily appears in the existence of the new "cargo dist host" subcommand, which
+was created to make "side-effectful networking" explicit, instead of riddling several random commands
+with various --dry-run flags.
+
+`host` takes several --steps:
+
+* create: ask Axo Releases to create hosting for the Apps we want to publish
+* upload: upload built Artifacts to the hosting that `create` made
+* release: create Releases for the hosted artifacts, making perma-urls like /v1.0.0/ live
+* announce: announce all the Releases, wiring them into "list all releases" and "latest release" endpoints
```

Comment:
unclear if latest is real- also unclear if it matters for this particular paragraph